docs/UI-TESTS.md

```diff
@@ -17,21 +17,23 @@ The following flows are covered/planned to be covered by UI tests. Tests that ar
     - [ ] Stats Today, This Week, This Month, This Year load
     - [ ] Tap chart on stats
 3. [Orders](../WooCommerce/WooCommerceUITests/Tests/OrdersTests.swift)
-    - [x] Orders list and single order screens load
-    - [x] Dismissal of order creation flow when tapping the Close button
-    - [ ] View product on single order screen
-    - [ ] Add customer note
-    - [ ] Add order note
-    - [ ] Update order status
-    - [ ] Issue refund
-    - [ ] Add shipping details
-    - Create a new order:
-       - [x] With a selected order status
-       - [ ] With a product
-       - [ ] With customer details
-       - [ ] With shipping
-       - [ ] With a fee
-       - [ ] With a customer note
+   - [x] Orders list loads
+   - [ ] Single Order Screen:
+     - [x] Single order screen loads
+     - [x] Dismissal of order creation flow when tapping the Close button
```

Comment:
Should this be another top-level checklist item? Since it's a separate test, unrelated to the single order screen.

Reply:
@rachelmcr did you mean the "Dismissal of order creation..." item?
If so I've committed a change that moves (and renames) this to the bottom of the section, as an item on the same level with "Orders list loads".
I'll go ahead an merge once the checks pass, but let me know if that's not what you meant.